src/runner/runner.go

```diff
@@ -69,10 +69,16 @@ func (r *Runner) Run() {
 		wg     sync.WaitGroup
 	)
 
+	newRawConfig, err := fetchConfig(r.configPaths, r.log)
+	if err != nil {
+		newRawConfig = r.backupConfig
+	}
+
 	for !stop {
-		newRawConfig, err := fetchConfig(r.configPaths, r.log)
+		newRawConfig, err = fetchConfig(r.configPaths, r.log)
 		if err != nil {
-			newRawConfig = r.backupConfig
+			newRawConfig = r.currentRawConfig
```

Comment:
Make this 
```
if r.currentRawConfig != nil {
    newRawConfig = r.currentRawConfig
} else {
    newRawConfig = r.backupConfig
}
```

This should do the trick you're aiming at.

Reply:
@jdoe7865623 thanks, addressed in https://github.com/Arriven/db1000n/pull/164/commits/4a6bb54185c7f158bb59ef6ab898d33f55a71616